mkdocs.yml

```diff
@@ -154,6 +154,7 @@ nav:
     - System: api/tests.system.md
     - Software: api/tests.software.md
     - SNMP: api/tests.snmp.md
+    - Security: api/tests.security.md
     - Routing:
```

Comment:
LEt's reorganize these via alphabetical order? (I know this is not only your PR .. just thinking about usability!)